index.js

```diff
@@ -0,0 +1,84 @@
+const db = require("./databaseFunctions.js");
+
+// TODO: Setup remote database connection instead of just locally hosted
+// TODO: check for internet connection to be able to grab items from database
+
+async function listTasks() {
+  await db
+    .getTasks()
+    .then((res) => {
+      if (res.length == 0) {
+        console.log("No tasks were found in database.");
+      } else {
+        res.forEach((task) => {
+          console.log(
+            `ID: ${task.id}, Task name: ${task.name}, Completion status: ${task.completed}`
+          );
+        });
+      }
+    })
+    .catch((err) => {
+      console.error("Error with tasks returned: " + err);
+    });
+}
+
+async function createTask(taskName) {
+  if (!taskName) {
+    console.error("Please enter a task name to create.");
+  }
+  if (typeof taskName !== "string") {
```

Comment:
I love this check, but I wonder if this should be an else/if rather than just an if - if the previous check succeeds (no task name is provided), do we need to check if it's a string?